virttest/tests/unattended_install.py

```diff
@@ -399,6 +399,16 @@ def answer_kickstart(self, answer_path):
             raise ValueError("Unexpected installation medium %s" % self.url)
         contents = re.sub(dummy_medium_re, content, contents)
 
+        dummy_guest_name_re = r'\bRH_GUEST_NAME\b'
+        if re.search(dummy_guest_name_re, contents):
+            guest_name = self.params.get("guest_name")
+            contents = re.sub(dummy_guest_name_re, guest_name, contents)
+
+        dummy_virt_module_stream_re = r'\bRH_VIRT_MODULE_STREAM\b'
+        if re.search(dummy_virt_module_stream_re, contents):
+            virt_module_stream = self.params.get("virt_module_stream", "")
+            contents = re.sub(dummy_virt_module_stream_re, virt_module_stream, contents)
```

Comment:
Same here.

Reply:
> Same here.

same with above,  this line is correct i think, since it use default `""`